android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/contract/HomeViewContract.java

```diff
@@ -0,0 +1,31 @@
+package com.liferay.mobile.formsscreenletdemo.contract;
+
+import com.liferay.apio.consumer.model.Thing;
+import com.liferay.mobile.formsscreenletdemo.view.HomeActivity;
+
+/**
+ * @author Luísa Lima
+ */
+public interface HomeViewContract {
+	interface HomeView {
+
+		void onSignOutCompleted();
+
+		void showDraftDialog();
+	}
+
+	interface HomePresenter {
+
+		void checkForDraft();
+
+		void loadDraft(Thing thing);
```

Comment:
It's better to be more specific when you are naming function, in this case, checkForFormRecordDraft and loadFormRecordDraft sound better.